app/src/main/java/de/westnordost/streetcomplete/osm/surface/SurfaceItem.kt

```diff
@@ -0,0 +1,90 @@
+    COMPACTED -> R.string.quest_surface_value_compacted
+    DIRT, SOIL, EARTH, MUD -> R.string.quest_surface_value_dirt
+    SETT, COBLLESTONE_FLATTENED -> R.string.quest_surface_value_sett
+    UNHEWN_COBBLESTONE -> R.string.quest_surface_value_unhewn_cobblestone
+    GRASS_PAVER -> R.string.quest_surface_value_grass_paver
+    WOOD -> R.string.quest_surface_value_wood
+    WOODCHIPS -> R.string.quest_surface_value_woodchips
+    METAL, METAL_GRID -> R.string.quest_surface_value_metal
+    GRAVEL -> R.string.quest_surface_value_gravel
+    PEBBLES -> R.string.quest_surface_value_pebblestone
+    GRASS -> R.string.quest_surface_value_grass
+    SAND -> R.string.quest_surface_value_sand
+    ROCK -> R.string.quest_surface_value_rock
+    CLAY -> R.string.quest_surface_value_clay
+    ARTIFICIAL_TURF -> R.string.quest_surface_value_artificial_turf
+    TARTAN -> R.string.quest_surface_value_tartan
+    PAVED_ROAD -> R.string.quest_surface_value_paved
+    UNPAVED_ROAD -> R.string.quest_surface_value_unpaved
+    GROUND_ROAD -> R.string.quest_surface_value_ground
+    PAVED_AREA -> R.string.quest_surface_value_paved
+    UNPAVED_AREA -> R.string.quest_surface_value_unpaved
+    GROUND_AREA -> R.string.quest_surface_value_ground
+    UNIDENTIFIED -> R.string.unknown_surface_title
+}
+
+val Surface.iconResId: Int get() = when (this) {
+    ASPHALT, CHIPSEAL -> R.drawable.surface_asphalt
+    CONCRETE -> R.drawable.surface_concrete
+    CONCRETE_PLATES -> R.drawable.surface_concrete_plates
+    CONCRETE_LANES -> R.drawable.surface_concrete_lanes
+    FINE_GRAVEL -> R.drawable.surface_fine_gravel
+    PAVING_STONES, PAVING_STONES_WITH_WEIRD_SUFFIX, BRICK, BRICKS -> R.drawable.surface_paving_stones
+    COMPACTED -> R.drawable.surface_compacted
+    DIRT, SOIL, EARTH, MUD -> R.drawable.surface_dirt
+    SETT, COBLLESTONE_FLATTENED -> R.drawable.surface_sett
+    UNHEWN_COBBLESTONE -> R.drawable.surface_cobblestone
+    GRASS_PAVER -> R.drawable.surface_grass_paver
+    WOOD -> R.drawable.surface_wood
+    WOODCHIPS -> R.drawable.surface_woodchips
+    METAL, METAL_GRID -> R.drawable.surface_metal
+    GRAVEL -> R.drawable.surface_gravel
+    PEBBLES -> R.drawable.surface_pebblestone
+    GRASS -> R.drawable.surface_grass
+    SAND -> R.drawable.surface_sand
+    ROCK -> R.drawable.surface_rock
+    CLAY -> R.drawable.surface_tennis_clay
+    ARTIFICIAL_TURF -> R.drawable.surface_artificial_turf
+    TARTAN -> R.drawable.surface_tartan
+    PAVED_ROAD -> R.drawable.path_surface_paved
+    UNPAVED_ROAD -> R.drawable.path_surface_unpaved
+    GROUND_ROAD -> R.drawable.surface_ground
+    PAVED_AREA -> R.drawable.surface_paved_area
+    UNPAVED_AREA -> R.drawable.surface_unpaved_area
+    GROUND_AREA -> R.drawable.surface_ground_area
+    UNIDENTIFIED -> R.drawable.background_fully_transparent // TODO - does not have the same size, maybe there is a fitting image?
```

Comment:
if we stay with empty image then I need to look into making it the same size as other images

Reply:
can't it be null?

Reply:
I meant the picture, that the picture is null.  But anyway,  not important

Reply:
with `null`
![screen07](https://user-images.githubusercontent.com/899988/218548043-d34396b0-749b-4c8a-a1c9-2498ba918278.png)

with transparent filler
![screen05](https://user-images.githubusercontent.com/899988/218548047-d0c67eb1-d371-404b-9291-e5a3c7cf551f.png)

Reply:
A, I see